genetics_ukbb/.bids-validator-config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ignore": [99, 15, 66]
+}
```

Comment:
This looks good to me @rwblair! Can we also have a `.bidsignore` in `genetics_ukbb` to ignore this file?

Reply:
Validator ignores dot files by default, it causing issues other places?

Reply:
true, see: https://github.com/bids-standard/bids-validator/issues/951